bool navigation_go
bool computer_vision_go
bool drive_roboteq_go
bool grabber_roboteq_go
bool nb1_go
bool nb2_go
bool nb3_go
bool servos_go
bool rr_go
bool avoidance_go
